modules/notify-privileged-activity/class-notify-privileged-activity.php

```diff
@@ -79,23 +111,25 @@ private static function send_notification( $user, $subject, $email_title, $templ
 				return;
 			}
 
-
 			\Automattic\VIP\Logstash\log2logstash(
 				[
 					'severity' => 'debug',
 					'feature'  => self::LOG_FEATURE_NAME,
 					'plugin'   => Constants::LOG_PLUGIN_NAME,
-					'message'  => 'User granted admin roles (notification type: ' . $template . ') user: ' . $user->user_login,
+					'message'  => 'User granted privileged roles (notification type: ' . $template . ') user: ' . $user->user_login,
 				]
 			);
-
-			Email::send( $user->ID, $admin_email, $subject, $template, [
+			$params = [
 				'user_login'  => $user->user_login,
 				'user_email'  => $user->user_email,
-				'user_role'   => 'Administrator',
+				'user_role'   => 'privileged-super-admin-granted' === $template ? 'Super Administrator' : 'Administrator',
 				'email_title' => $email_title,
 				'admin_url'   => admin_url(),
-			] );
+			];
+			if ( is_multisite() ) {
```

Comment:
Very minor: We might be able to simplify this by always using `network_admin_url`. See https://github.com/WordPress/wordpress-develop/blob/6.8.1/src/wp-includes/link-template.php#L3822